pressbooks/pressbooks-book
# ToC Toggle Button A11y Fix

This PR fixes parsing validation issues with our ToC toggle button by changing aria-expanded to data-expanded and adjusting the relevant JS and CSS.

To test:
1. create a book that has a two-level ToC and at least one H1 in a chapter
2. visit the home page of that book and use the ToC toggle and confirm the show/hide all contents button works as expected
https://user-images.githubusercontent.com/13485451/139470618-23544ec2-d07f-4af6-8a0b-01ef84468fe3.mp4
3. check page source for validation issues (https://validator.w3.org/nu/#textarea) and confirm they have been removed
